.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+
+test
```

Comment:
- [ ] As per the project requirement kindly make sure the test.md file doesn't appear in GitHub, for that you have to write a test.md instead of test only :recycle: